spec/features/beis_users_can_upload_level_b_activities_spec.rb

```diff
@@ -42,7 +42,8 @@
       "Aid type",
       "Free Standing Technical Cooperation",
       "Aims/Objectives",
-      "NF Partner Country PO"
+      "NF Partner Country PO",
+      "Comments"
```

Comment:
Might be worth adding a step to the expectations below to click on the "Comments" tab and see the comment? I know we don't test this for everything else, but it'd be good to capture as documentation (particularly as this was behaviour we didn't really know we got for free!)